src/time/format.cr

```diff
@@ -28,6 +28,7 @@
 # * **%k**: hour of the day, 24-hour clock, blank padded (" 0", " 1", ..., "24")
 # * **%l**: hour of the day, 12-hour clock, blank padded (" 0", " 1", ..., "12")
 # * **%L**: milliseconds, zero padded (000, 001, ..., 999)
+# * **%N**: nanoseconds, zero padded (000000000, 000000001, ...)
```

Comment:
Please, add the last value after the ellipsis to mark complete range, like in the other examples.

Reply:
Thanks, @Sija.